templates/nodeovc/nodeovc_test.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase
+from unittest.mock import MagicMock, patch, PropertyMock
+import tempfile
+import shutil
+import os
+import pytest
+
+from js9 import j
+from zerorobot.template.base import TemplateBase
+from nodeovc import Nodeovc
+
+class TestNodeovc(TemplateBase):
```

Comment:
Why is the a child of TemplateBase ?
a test should inherits from `unittest.TestCase` class